### DISCUSSION TITLE
Is it support <image> tag ?

### DISCUSSION THREAD

**Discussion by octalflow** [litehtml/litehtml#373] [Q&A]
I tried but failed.
Is it support <image> tag ?

thank you.

**Comment by tordex** [comment]
I guess the right tag is ```<img>``` isn't it?
```<img>``` is supported.

**Comment by octalflow** [comment]
No,
not img tag but image tag.

**Comment by tordex** [comment]
Sorry, what?

**Comment by octalflow** [comment]
image tag.
Not img tag.

Do you understand me?

**Comment by octalflow** [comment]
sample tag is

\<image width="1500" height="2100" xlink:href="../Images/cover.jpg" /\>

**Reply by tordex** [reply]
This tag is not supported. By "xlink:href" I see this is something from SVG, right? litehtml doesn't support SVG.

**Reply by octalflow** [reply]
I see. thank you.
